app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Issue: The variable name 'people' is not descriptive enough. Consider renaming it to something more descriptive like 'person_registry' to better convey its purpose.